### ISSUE TITLE
When the module is active, the order reference code disappears

### ISSUE THREAD

**Issue by ptrolese** [Matt75/displayordercarrier#5]
**Describe the bug**
Hello!! I have installed the module and it works perfectly but when it is installed in the order list the order reference code disappears, as in the screenshot.

**Screenshots**
![download](https://user-images.githubusercontent.com/88393095/128046387-c0998b08-495e-43b9-a7dd-30e208e847f8.png)

**Additionnal information**
PrestaShop version: 1.7.6.7
PHP version: 7.3

**Comment by Matt75** [comment]
Cannot reproduce issue, maybe a conflict with another module installed.

Moreover, your PrestaShop version 1.7.6.7 is not compliant with PHP 7.3 : https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/#php-compatibility-chart

**Comment by Urban225** [comment]
I have exactly the same problem with Prestashop 1.7.6.4 and php 7.2.34.

**Comment by Matt75** [comment]
@Urban225 Please provide more info, did you have another module to customize this listing ?

**Comment by Urban225** [comment]
@Matt75  

Yes, I have the Mirakl and Colissimo module that adds information: 

https://addons.prestashop.com/fr/marketplace/24950-mirakl-connecteur-multi-marketplaces.html

https://addons.prestashop.com/fr/transporteurs/42138-colissimo-officiel-vos-expeditions-avec-colissimo.html

![Capture](https://user-images.githubusercontent.com/55872821/136551378-025cb153-91a4-4028-ac42-8e10f9ab781e.JPG)

**Comment by Matt75** [comment]
@Urban225 thanks, I will check this

**Comment by ytilotti** [comment]
Same issue here.

Need to replace `$params['select'] .= ', a.id_carrier, car.id_reference AS carrier_reference, car.name AS carrier_name';` by `$params['select'] .= ', a.id_carrier, car.name AS carrier_name';`

**Comment by Urban225** [comment]
@ytilotti It's perfect :)

It works well, thank you!
